challenge.rb

```diff
@@ -1,26 +1,25 @@
-
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map{|element| element.capitalize}
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.select{|element| element=="dog"}
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.find_all{|element| element != "dog"}
```

Comment:
Check the `reject` method. Its nice to phrase the code positively, when you can.